src/js/modules/components/moveable.js

```diff
@@ -1,15 +1,26 @@
 export const MOVE_END = 'moveend';
```

Comment:
I know, that it is not in scope of this PR but i'm thinking that maybe better name for the whole functionality will be `draggable`? What do You think?